examples/javascript/test/browser/chromeSpecificCaps.spec.js

```diff
@@ -47,6 +47,17 @@ suite(function (env) {
       await driver.quit();
     });
 
+    it('Add Extension', async function () {
+      const options = new Chrome.Options();
+      let driver = await env
+        .builder()
+        .setChromeOptions(options.addExtensions(['/path/to/extension.crx']))
+        .build();
+
+      await driver.get('https://www.google.com');
```

Comment:
we're moving all url's in code samples to "https://www.selenium.dev/" (or) https://www.selenium.dev/selenium/web/ mostly!